### PR TITLE
label update health function

### PR DESCRIPTION
**Pull request by zelosos**

### CODE REVIEW

**Comment by zelosos** [comment]
I will redo the comments of the wRAM with a new pull request

**Comment by kemenaran** [comment]
Nice piece of work 👌 

I'll read it and merge it later today, but at first sight it looks good to me.

File: src/code/bank2.asm [hunk 1]
```diff
@@ -4787,119 +4787,153 @@ LoadRupeesDigits::
...
     dec  a                                        ; $63A9: $3D
     ld   [wHasMedicine], a                        ; $63AA: $EA $0D $DB
-    ld   a, $08                                   ; $63AD: $3E $08
+    ; add one heart to the $wHealth
+    ld   a, ONE_HEART                                   ; $63AD: $3E $08
     ld   [wHealth], a                             ; $63AF: $EA $5A $DB
+    ; add 16 hearts to the wAddHealthBuffer
     ld   a, [wAddHealthBuffer]                    ; $63B2: $FA $93 $DB
-    add  $80                                      ; $63B5: $C6 $80
+    add  ONE_HEART*16                                      ; $63B5: $C6 $80
```
Comment (kemenaran) [inline src/code/bank2.asm:4931]: 👍

File: src/code/bank2.asm [hunk 2]
```diff
@@ -4787,119 +4787,153 @@ LoadRupeesDigits::
...
     ld   [wAddHealthBuffer], a                    ; $63B7: $EA $93 $DB
-    ld   a, $A0                                   ; $63BA: $3E $A0
+    ; make player invincible so no deamage can be taken some time
```
Comment (kemenaran) [inline src/code/bank2.asm:4933]:
```suggestion
    ; make player invincible so no damage can be taken some time
```

File: src/code/bank2.asm [hunk 3]
```diff
@@ -4787,119 +4787,153 @@ LoadRupeesDigits::
...
+    ld   a, DEMAGE_COOLDOWN_TIME                                   ; $63BA: $3E $A0
```
Comment (kemenaran) [inline src/code/bank2.asm:4934]:
```suggestion
    ld   a, DAMAGE_COOLDOWN_TIME                                   ; $63BA: $3E $A0
```

File: src/constants/gameplay.asm [hunk 4]
```diff
@@ -239,3 +239,9 @@ FISHING_GAME_PRICE        equ 10
 ; Fishing game rewards
 FISHING_GAME_SMALL_REWARD equ 05
 FISHING_GAME_LARGE_REWARD equ 20
+
+; one heart equals XX health
+ONE_HEART equ $08
+
+; how much time has to pass until the player can receive damage again
+DEMAGE_COOLDOWN_TIME equ $A0
```
Comment (kemenaran) [inline src/constants/gameplay.asm:247]:
```suggestion
DAMAGE_COOLDOWN_TIME equ $A0

```

File: src/constants/sfx.asm [hunk 5]
```diff
@@ -253,3 +253,5 @@ NOISE_SFX_BOOMERANG               equ $2D;probably boomerang
 NOISE_SFX_EAGLE_TOUCHDOWN         equ $30;probably eagle wings
 NOISE_SFX_EAGLE_LIFT_UP           equ $31;also eagle probably
 ; entries to be added to at least $AA
+
+LOW_HEALTH_SFX_PAUSE              equ $30 ; pause beetween low health SFX peeps
```
Comment (kemenaran) [inline src/constants/sfx.asm:257]:
```suggestion
LOW_HEALTH_SFX_PAUSE              equ $30 ; pause beetween low health SFX beeps
```

**Comment by kemenaran** [comment]
Thanks!

**Pull request merged**